api/container/containerv2/worker_pool.go

```diff
@@ -6,6 +6,20 @@ import (
 	"github.com/IBM-Cloud/bluemix-go/client"
 )
 
+type WorkerPoolConfig struct {
+	DiskEncryption         bool                    `json:"diskEncryption,omitempty"`
+	Entitlement            string                  `json:"entitlement"`
+	Flavor                 string                  `json:"flavor"`
+	HostPool               string                  `json:"hostPool,omitempty"`
```

Comment:
please rename the field to try to keep it consistent
```suggestion
	HostPoolID               string                  `json:"hostPool,omitempty"`
```